app/javascript/pages/admin/Events/index.js

```diff
@@ -19,6 +19,8 @@ import s from './main.css'
 
 import 'style-loader!css-loader!react-table/react-table.css'
 
+const eventsSort = 'STARTS_AT_ASC'
```

Comment:
Shouldn't this be STARTS_AT_DESC?

Reply:
🤦‍♂️

Reply:
It seems like it would also be possible to resize all headings client-side by removing `sortable: false` from [here](https://github.com/zendesk/volunteer_portal/blob/7b4d8b1d85bba0fb65a0eac1c76219b4c2ac4dfa/app/javascript/pages/admin/Events/index.js#L51). Not sure why it was marked as `false` in the first place. @craig-day @roseleaf may know?

Reply:
@craig-day  or @roseleaf I would also like to know the answer to @danielbreves 's question ☝️